packages/cli/src/commands/lint.ts

```diff
@@ -127,6 +127,11 @@ function lintConfigCallback(argv: LintOptions, version: string) {
     return;
   }
 
+  if (argv.format === 'json') {
```

Comment:
Is this part covered with a test? If not, it's better to add one.

Reply:
Added an e2e test.